sycl/source/detail/config.hpp

```diff
@@ -176,6 +176,85 @@ template <> class SYCLConfig<SYCL_PI_TRACE> {
   }
 };
 
+template <> class SYCLConfig<SYCL_PARALLEL_FOR_RANGE_ROUNDING_TRACE> {
+  using BaseT = SYCLConfigBase<SYCL_PARALLEL_FOR_RANGE_ROUNDING_TRACE>;
+
+public:
+  static bool get() {
+    static bool Initialized = false;
+    static bool Trace = false; // No tracing by default
+
+    // Check env var setting for tracing only once
+    if (Initialized)
+      return Trace;
+
+    const char *ValStr = BaseT::getRawValue();
+    Trace = ValStr != nullptr;
+    Initialized = true;
+    return Trace;
+  }
+};
+
+template <> class SYCLConfig<SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING> {
+  using BaseT = SYCLConfigBase<SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING>;
+
+public:
+  static bool get() {
+    static bool Initialized = false;
+    static bool DisableRounding = false; // Range rounding on by default
+
+    // Check env var setting for rounding only once
+    if (Initialized)
+      return DisableRounding;
+
+    const char *ValStr = BaseT::getRawValue();
+    DisableRounding = ValStr != nullptr;
+    Initialized = true;
+    return DisableRounding;
```

Comment:
```suggestion
    static const char *ValStr = BaseT::getRawValue();
    return ValStr != nullptr;
```

Reply:
See comment above.
For the same reason, the code as written is necessary to avoid calling getenv multiple times.